addons/l10n_in/report/account_invoice_report.py

```diff
@@ -259,14 +257,12 @@ def _from(self):
 
     def _where(self):
         return """
-                WHERE am.state = 'posted'
-                    AND tag_rep_ln.account_tax_report_line_id in (SELECT res_id FROM ir_model_data WHERE module='l10n_in' AND name in ('tax_report_line_igst','tax_report_line_cgst','tax_report_line_sgst'))
+                WHERE tag_rep_ln.account_tax_report_line_id in (SELECT res_id FROM ir_model_data WHERE module='l10n_in' AND name in ('tax_report_line_igst','tax_report_line_cgst','tax_report_line_sgst'))
```

Comment:
Why did you remove the posted?

Reply:
because now we give filter on the report that all_entery or posted only.